src/generators/crud/default/views/view.php

```diff
@@ -98,20 +99,24 @@
     <hr />
 
     <?php
-    echo "<?php \$this->beginBlock('{$generator->modelClass}'); ?>\n";
+    echo "<?php \$this->beginBlock('".substr(strrchr($model::className(), '\\'), 1)."'); ?>\n";
```

Comment:
But this is just an internal array key for preparing content via blocks, there's no output of this value. And you loose information, could be a problem to have two models with the same name in different namespaces.